src/lib.rs

```diff
@@ -85,7 +88,8 @@ pub struct NetworkResource {
 #[cfg(not(target_arch = "wasm32"))]
 #[allow(dead_code)] // FIXME: remove this struct?
 struct ServerListener {
-    receiver_task: bevy_tasks::Task<()>, // needed to keep receiver_task alive
+    receiver_task: bevy_tasks::Task<()>,
```

Comment:
This wrap changes the meaning of the comment.

Reply:
oh, good catch 👍